pythonpro/modules/tests/test_module_index_view_roles_access.py

```diff
@@ -0,0 +1,189 @@
+import pytest
+from django.core.management import call_command
+from django.urls import reverse
+from model_bakery import baker
+from rolepermissions.roles import assign_role, clear_roles
+
+from pythonpro.django_assertions import dj_assert_contains, dj_assert_not_contains
+from pythonpro.modules import facade
+
+
+@pytest.fixture
+def modules(transactional_db):
+    call_command('loaddata', 'pythonpro_modules.json')
+    modules = facade.get_all_modules()
+    return modules
+
+
+@pytest.fixture
+def resp_lead_user(client, django_user_model, modules):
```

Comment:
You need only use "client_with_lead" and you will get a client with a lead logged.